include/deal.II/matrix_free/operators.h

```diff
@@ -288,6 +288,48 @@ namespace MatrixFreeOperators
 
 
 
+  /**
+   * This class implements the operation of the action of a mass matrix
+   * and additionally the action of a lumped mass matrix that can be used for
```

Comment:
i would reword the second part. The class does not implement an action of lumped mass matrix, in that you can't do `vmult` with it. Instead, the diagonal is computed by lumping all elements, which is one of many possible approaches. I would just move this info to `compute_diagonal()` and not mention it here.